### ISSUE TITLE
Support Changing Method Inflectors

### ISSUE THREAD

**Issue by rdohms** [thephpleague/tactician-bundle#3]
We should be able to support changing the method Inflector to use.

```yml
tactician:
    default: 
        method_inflector: "tactician.handler.inflector.handle"
```

**Comment by rdohms** [comment]
So this actually needs to be on the top level, so:

```yaml
tactician:
        method_inflector: "tactician.handler.inflector.handle"
```

**Comment by theUniC** [comment]
Currently it's not possible to specify a custom inflector. As this feature is implemented at a Service Container extension level, it can only see its own registered services.

**https://github.com/thephpleague/tactician-bundle/blob/master/DependencyInjection/TacticianExtension.php#L70**

So then it's only possible to change the inflector strategy by one defined by the bundle. In order to be able to define a custom inflector strategy provided by other bundles I think this should be moved from the bundle extension to a compiler pass.

If you're OK with this, I can do it myself and open a PR.

Thanks!
Christian.

**Comment by rosstuck** [comment]
That would be great! @rdohms and I are both at a conference this weekend so
we might be a little slower responding but a PR would be excellent :)

**Comment by boekkooi** [comment]
Are you sure a custom compiler pass is the best solution here? Since the problem that I see is that the `$container->has` should not be used since the DI compile step will take care of checking if the service exists.

So currently I see one bug in this issue and one enhancement.
1. The bug being a unneeded en disrupting container has service check (mentioned by @theUniC)
2. The enchantment being that `[method_inflector](https://github.com/thephpleague/tactician-bundle/blob/adad4ce89371b18997519bbc9580f45a8e43ba59/DependencyInjection/Configuration.php#L56)` should be moved to `[commandbus.<name>.method_inflector](https://github.com/thephpleague/tactician-bundle/blob/adad4ce89371b18997519bbc9580f45a8e43ba59/DependencyInjection/Configuration.php#L31)`

Personally I see no reason for a compiler pass.

**Comment by theUniC** [comment]
Agreed. Indeed a CompilerPass for this is unnecessary and it would just be a matter of removing the check.

**Comment by rosstuck** [comment]
Closed by #15
